### PR TITLE
feat: scan content with Blockaid API

### PR DESCRIPTION
**Pull request by bonustrack**
This PR add Blockaid API to scan content of a proposal to detect malicious site automatically.

### CODE REVIEW

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by ChaituVR** [comment]
Everything else works like well  🪄

File: src/writer/proposal.ts [hunk 1]
```diff
@@ -64,6 +65,20 @@ export async function verify(body): Promise<any> {
...
 
+  try {
+    const content = `
+      ${msg.payload.name || ''}
+      ${msg.payload.body || ''}
+      ${msg.payload.discussion || ''}
+    `;
+
+    if (await isMalicious(content)) {
+      return Promise.reject(`invalid proposal content`);
```
Comment (ChaituVR) [inline src/writer/proposal.ts:76]: Maybe 
```suggestion
      return Promise.reject(`malicious proposal content`);
```

File: src/helpers/blockaid.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import fetch from 'node-fetch';
+
+const BLOCKAIDS_API_URL = 'https://api.blockaid.io/dapp/v1/scan';
+const BLOCKAIDS_API_KEY = 'knUtyWFpcba5LlMUsky1jUkMM7kKPTug';
```
[the remark this answers is not in the document]
Comment (bonustrack) [reply src/helpers/blockaid.ts:4]: Hope so

File: src/writer/proposal.ts (hunk 1 above)
Comment (bonustrack) [reply src/writer/proposal.ts:76]: I'd rather stay vague, we may catch scammer asking in our support why his proposal is rejected.

**Comment by bonustrack** [comment]
Why test failing?

**Review by ChaituVR — APPROVED** [review]
tAck

**Comment by ChaituVR** [comment]
These tests fail randomly. will take a look. but feel free to merge

**Pull request merged**